test.js

```diff
@@ -211,5 +211,5 @@ test('prevent setting/getting `__proto__`', t => {
 	dotProp.set({}, '__proto__.unicorn', '🦄');
 	t.not({}.unicorn, '🦄'); // eslint-disable-line no-use-extend-native/no-use-extend-native
 
-	t.is(dotProp.get({}, '__proto__'), undefined);
+	t.false(dotProp.get({}, '__proto__', false));
```

Comment:
Can you do a new test (with a descriptive title) instead of editing an existing one? I think it would also be clearer if you did something like:

```js
t.is(dotProp.get({}, 'foo', '🦄'), '🦄');
```